src/SystemCommands-SourceCodeCommands/SycExtractTempCommand.class.st

```diff
@@ -32,11 +32,16 @@ SycExtractTempCommand >> applyResultInContext: aSourceCodeContext [
 { #category : 'converting' }
 SycExtractTempCommand >> asRefactorings [
 
-	^ {RBExtractToTemporaryRefactoring
-		  extract: sourceNode sourceInterval
-		  to: tempName
-		  from: method selector
-		  in: method origin}
+	| driver |
+	driver := ReExtractToTemporaryDriver new.
+	driver 
+		extract: sourceNode sourceInterval
+	 	to: tempName
+	 	from: method selector
+	 	in: method origin.
+		
+	^ driver runRefactoring.
+
 ]
```

Comment:
Take a look at how were other drivers configured to run here. This does work, but there is a slightly better way:
- override `isComplexRefactoring` to return false
- this method is unused then
- override `execute` with this code
Take a look at `SycRemoveClassCommand`. I'm saying this since `asRefactoring` and `isComplexRefactoring` will be removed in the future and this will then not work.